api/src/main/java/org/pmiops/workbench/impersonation/ImpersonatedWorkspaceServiceImpl.java

```diff
@@ -197,24 +204,16 @@ public void deleteOrphanedRawlsWorkspace(
       throw new ServerErrorException(e);
     }
 
-    // TODO
-    // I can't figure out how to make this work from a tool
```

Comment:
The app and crons should be able to do this.  So it's now a flag: try to delete billing projects or not